### DISCUSSION TITLE
Provide Trivy as a Conan package

### DISCUSSION THREAD

**Discussion by lukas-braune** [aquasecurity/trivy#8064] [Ideas]
### Description

Please provide Trivy as a Conan package on [Conan Center](https://conan.io/center). This will simplify the integration of Trivy into projects using Conan, significantly reducing setup time and effort for developers.

### Target

None

### Scanner

None

**Comment by itaysk** [comment]
I thought Conan is a package manger for C/C++ pacakges (meaning for libraries used in code), not for compiled tools. Am I wrong?

**Reply by roalter** [reply]
Well, indeed that is the case.

If security scans are done during build and trivy is used it might make sense to have it built directly from latest.

Compare it to cmake; it is a so-called build requirement.

But I think this might be more a pull request for Conan Center index?

**Reply by itaysk** [reply]
I think the community can add trivy to other package mangers if there's a need.
